File: Exercicios-Beecrowd/C#/1009.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Teste
{
    internal class Program
    {
        static void Main(string[] args)
        {
            string Z = Convert.ToString(Console.ReadLine());
            double a = Convert.ToDouble(Console.ReadLine());
            double b = Convert.ToDouble(Console.ReadLine());
            double soma = a + (b * 0.15);
            string resultado = soma.ToString("F2");
            Console.WriteLine("TOTAL = R$ {0}",resultado);
            


        }
    }
}

File: Exercicios-Beecrowd/Kotlin/1009.kt
import java.util.*

fun main(args: Array<String>) {

	    val nome= readLine()
    val valor1= readLine()
    val valor2= readLine()


    if (valor1 != null && valor2!=null){
        

        val nota1=valor1.toDouble()
        val nota2=valor2.toDouble()

        val resultado=(nota2-(nota2*0.85))+nota1
        val formatado = ("%.2f".format(resultado))
        println("TOTAL = R$ ${formatado}")
    }
}